subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -42,6 +42,7 @@ config BT_FAST_PAIR_PN
 rsource "fp_crypto/Kconfig.fp_crypto"
 rsource "fp_storage/Kconfig.fp_storage"
 rsource "fmdn/Kconfig"
+rsource "use_case/Kconfig"
```

Comment:
Consider keeping it at the top of the file (above `CONFIG_BT_FAST_PAIR_SUBSEQUENT_PAIRING`) as this is a top-level configuration that should be highlighted I think